### PR TITLE
SNS topic for reports

### PR DESCRIPTION
**Pull request by airmonitor**
- Added SNS topic for sending Inspector reports to subscribers. Next PR will contain integration with https://github.com/awslabs/amazon-inspector-finding-forwarder

- Ruleset arn's moved to separate dict in locals, this way we can select particular arn based on region provided by data "aws_region"

### CODE REVIEW

**Comment by FnTm** [comment]
@airmonitor Thank you for this addition! I'd be really interested in using this. Do you maybe have any idea on who we should ping to get this pr reviewed and moving again?

File: main.tf [hunk 1]
```diff
@@ -4,28 +4,47 @@ terraform {
...
   }
 }
+
+data "aws_region" "current_region" {}
+data aws_caller_identity "current_identity" {}
+
 locals {
   enabled_count   = var.enabled ? 1 : 0
   scheduled_count = var.enable_scheduled_event && var.enabled ? 1 : 0
+  ruleset_cis_arn = {
```
Comment (bushong1) [inline main.tf:14]: Love these ARN lookup  changes, thanks!

File: main.tf [hunk 2]
```diff
@@ -44,7 +63,7 @@ resource "aws_inspector_assessment_template" "assessment" {
 
 resource "aws_iam_role" "inspector_event_role" {
   count = local.scheduled_count
-  name  = "${var.name_prefix}-inspector-event-role"
+  name  = "${var.name_prefix}-inspector-event"
```
Comment (bushong1) [inline main.tf:66]: This would be a breaking change to existing users, and a major version bump, so we'd prefer not to change names for cosmetic reasons.

**Review by bushong1 — CHANGES_REQUESTED** [review]
All the SNS Topic subscription features really feels like an external module to me.

We're happy to accept changes to the ARN lists, as well as exporting any resources you need to access.

File: README.md [hunk 3]
```diff
@@ -5,6 +5,7 @@ A terraform module to deploy [Amazon Inspector](https://docs.aws.amazon.com/insp
 ## Prerequisites
 
 * Amazon Inspector Agent [installed](https://docs.aws.amazon.com/inspector/latest/userguide/inspector_installing-uninstalling-agents.html#install-linux) on desired EC2 instances.
+* Inspector Rule Set ARN's proper to AWS Region - [list](https://docs.aws.amazon.com/inspector/latest/userguide/inspector_rules-arns.html#eu-west-1)
```
Comment (mattjorg) [inline README.md:8]: Remove the `#eu-west-1` portion of the link

**Review by mattjorg — CHANGES_REQUESTED** [review]

**Pull request closed without merging**